frontend/apps/api_users/logging_helpers.py

```diff
@@ -0,0 +1,99 @@
+import csv
+import logging
+from dataclasses import dataclass
+from io import StringIO
+
+import boto3
+from django.conf import settings
+
+logger = logging.getLogger()
+
+
+@dataclass
+class APIKeyForLogging:
+    key: str
+    key_name: str
+    user_name: str
+    email: str
+    usage_reason: str
+
+    @property
+    def dc_environment(self):
+        return getattr(settings, "DC_ENVIRONMENT", None)
+
+    @property
+    def file_name(self):
```

Comment:
We might want to use `slugify` here? Are we sure none of these can contain `/`, for example?